### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by WilliamsEfe** [community/community#55991] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

What are some of the limitations of github copilot?

**Comment by Leke-Ariyo** [comment]
Vulnerability to bias and limited support for non-coding tasks

**Comment by tuves** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answering and locking the post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
